Formula/redis.rb

```diff
@@ -62,6 +62,7 @@ def plist; <<-EOS.undent
         <array>
           <string>#{opt_bin}/redis-server</string>
           <string>#{etc}/redis.conf</string>
+          <string>--daemonize no</string>
```

Comment:
Not sure, made sense to keep them together as these should never be separated, but I don't know what's the preferred style… quick search didn't reveal any established standards.